lib/paf.ml

```diff
@@ -206,8 +206,7 @@ end = struct
         | `Close ->
             Log_server.debug (fun m -> m "next read operation: `close") ;
             Lwt.wakeup_later notify_rd_exit () ;
-            flow.Easy_flow.rd_closed <- true ;
```

Comment:
oh here we can get then rid of the rd_closed / wr_closed!? maybe not -- but maybe these fields should then still be set?